ext/libnativemath/libnativemath.cpp

```diff
@@ -48,6 +48,6 @@ int native_eval(string input, char *value, char *value_type) {
 
   value_type[0] = ans.GetType();
 
-  strcpy(value, ans.AsString().c_str());
+  strcpy(value, ans.ToString().c_str());
```

Comment:
```suggestion
  strcpy(value, ans.AsString().c_str());
```

I opened a [PR](https://github.com/niltonvasques/equations-parser/pull/42) with a better solution in the `equations-parser` repository. Can you review and test the solution here?